File: backend/src/main/java/Entity/Student.java
package Entity;

import jakarta.persistence.*;

@Entity
@Table(name = "student")
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "student_id")
    private Long studentId;

    private String mssv;
    private String name;
    private String email;
    private String phone;

    @ManyToOne
    @JoinColumn(name = "class_id")
    private Classroom classRoom;
}

File: frontend/src/models/student.js
export function mapStudentListItem(item={})
{
  return {
    studentId: item.studentId ?? item.id ?? item.student_id ?? null,
    fullName: item.fullName ?? item.name ?? item.full_name ?? '',
    email: item.email ?? item.emailAddress ?? item.email_address ?? '',
  };
}
export function buildStudentCreateRequest(students = []) {
  return {
    students: students.map((s) => ({
      studentId: s.mssv || s.studentId || '',
      fullName: s.name || s.fullName || '',
      email: s.email || '',
    })),
  };
}